coremltools/converters/mil/frontend/torch/test/testing_utils.py

```diff
@@ -249,6 +255,7 @@ def run_compare_torch(
         atol=1e-04,
         rtol=1e-05,
         input_as_shape=True,
+        input_dtype=np.float32,
```

Comment:
same here.
Please consider adding it to the end.

Reply:
Same here, luckily this time everything is good I think
1. CI is green, so nothing is broken inside our test
2. This is not a public API, so nothing outside should get affected